### DISCUSSION TITLE
Convergence Issues with New Parameters

### DISCUSSION THREAD

**Discussion by alanchavez86** [idaholab/moose#16384] [Q&A Modules: Phase field] [answered]
Hi all!

I currently have an input file (FeCrCo.i) used for the phase field simulation of the decomposition of FeCrCo alloy. The input file works correctly using parameters from Koyama 2004. When switching my parameters to those in Lv. 2010, I'm running into convergence issues where my data doesn't go past 1 second. I changed my temperature to a global parameter and now I'm looking into my executioner block to see if that's where my issue is.

Any help is appreciated. Thank you!
[FeCrCo.zip](https://github.com/idaholab/moose/files/5639815/FeCrCo.zip)
[Koyama_2004.pdf](https://github.com/idaholab/moose/files/5639822/Koyama_2004.pdf)
[Lv_2010_PF of FeCrCo.pdf](https://github.com/idaholab/moose/files/5639824/Lv_2010_PF.of.FeCrCo.pdf)

**Comment by laagesen** [comment, marked as answer]
Here are a couple suggestions to troubleshoot this:
-Make sure that you are using the same units when you change to the new set of parameters
-Check each parameter that you are changing and make sure that it is the same order of magnitude as the original parameter. Historically the guidance has always been that the solvers will work best when units are chosen such that all parameters are of order unity. That is why I mentioned making sure the units are the same between the parameters in the two papers. 
-A recently added option is automatic variable scaling, which will apply a scaling factor to each residual equation to try to make sure that the contribute approximately the same to the residual. This option can be turned on by setting the parameter `automatic_scaling = true` in the `Executioner` block.
